src/mono/netcore/sample/Android/Program.csproj

```diff
@@ -57,8 +57,9 @@
       <AdbTool>$(ANDROID_SDK_ROOT)\platform-tools\adb</AdbTool>
     </PropertyGroup>
     <Error Condition="'$(ANDROID_SDK_ROOT)' == ''" Text="ANDROID_SDK_ROOT should point to a valid Android SDK location" />
+    <Message Importance="High" Text="Uninstalling app if it exists (throws an error if it doesn't but it can be ignored):"/>
     <Exec Command="$(AdbTool) uninstall net.dot.HelloAndroid" ContinueOnError="WarnAndContinue" />
-    <Exec Command="$(AdbTool) install -r bin/bundle/bin/HelloAndroid.unaligned.apk" />
+    <Exec Command="$(AdbTool) install -r bin/bundle/bin/HelloAndroid.apk" />
```

Comment:
nit: you probably don't need `-r` (reinstall) option since you already uninstalled it before.